src/test/scala/example/ScalaJSExampleTest.scala

```diff
@@ -1,19 +1,16 @@
 package example
 
-import scala.scalajs.js
-import js.Dynamic.{ global => g }
-import scala.scalajs.test.JasmineTest
+import utest._
 
-object ScalaJSExampleTest extends JasmineTest {
+object ScalaJSExampleTest extends TestSuite {
 
-  describe("ScalaJSExample") {
+ import ScalaJSExample._
```

Comment:
There are indentation issues, here. 1 space instead of 2. Same with the following lines.